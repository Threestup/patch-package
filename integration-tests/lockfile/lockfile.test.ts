import { runIntegrationTest } from "../runIntegrationTest"
runIntegrationTest({ projectName: "lockfile", shouldProduceSnapshots: true })
