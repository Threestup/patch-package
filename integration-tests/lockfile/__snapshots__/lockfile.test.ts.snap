// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test lockfile: check patch with lockfile 1 1`] = `
"SNAPSHOT: check patch with lockfile 1
diff --git a/node_modules/left-pad/index.js b/node_modules/left-pad/index.js
index e90aec3..7f8fdaa 100644
--- a/node_modules/left-pad/index.js
+++ b/node_modules/left-pad/index.js
@@ -22,9 +22,9 @@ var cache = [
 function leftPad (str, len, ch) {
   // convert \`str\` to a \`string\`
   str = str + '';
-  // \`len\` is the \`pad\`'s length now
+  // \`len\` is the \`npm\`'s length now
   len = len - str.length;
-  // doesn't need to pad
+  // doesn't need to npm
   if (len <= 0) return str;
   // \`ch\` defaults to \`' '\`
   if (!ch && ch !== 0) ch = ' ';
@@ -32,12 +32,12 @@ function leftPad (str, len, ch) {
   ch = ch + '';
   // cache common use cases
   if (ch === ' ' && len < 10) return cache[len] + str;
-  // \`pad\` starts with an empty string
-  var pad = '';
+  // \`npm\` starts with an empty string
+  var npm = '';
   // loop
   while (true) {
-    // add \`ch\` to \`pad\` if \`len\` is odd
-    if (len & 1) pad += ch;
+    // add \`ch\` to \`npm\` if \`len\` is odd
+    if (len & 1) npm += ch;
     // divide \`len\` by 2, ditch the remainder
     len >>= 1;
     // \\"double\\" the \`ch\` so this operation count grows logarithmically on \`len\`
@@ -47,6 +47,6 @@ function leftPad (str, len, ch) {
     // \`len\` is 0, exit the loop
     else break;
   }
-  // pad \`str\`!
-  return pad + str;
+  // npm \`str\`!
+  return npm + str;
 }
END SNAPSHOT"
`;

exports[`Test lockfile: check patch with lockfile 2 1`] = `
"SNAPSHOT: check patch with lockfile 2
diff --git a/node_modules/left-pad/index.js b/node_modules/left-pad/index.js
index e90aec3..7f8fdaa 100644
--- a/node_modules/left-pad/index.js
+++ b/node_modules/left-pad/index.js
@@ -22,9 +22,9 @@ var cache = [
 function leftPad (str, len, ch) {
   // convert \`str\` to a \`string\`
   str = str + '';
-  // \`len\` is the \`pad\`'s length now
+  // \`len\` is the \`npm\`'s length now
   len = len - str.length;
-  // doesn't need to pad
+  // doesn't need to npm
   if (len <= 0) return str;
   // \`ch\` defaults to \`' '\`
   if (!ch && ch !== 0) ch = ' ';
@@ -32,12 +32,12 @@ function leftPad (str, len, ch) {
   ch = ch + '';
   // cache common use cases
   if (ch === ' ' && len < 10) return cache[len] + str;
-  // \`pad\` starts with an empty string
-  var pad = '';
+  // \`npm\` starts with an empty string
+  var npm = '';
   // loop
   while (true) {
-    // add \`ch\` to \`pad\` if \`len\` is odd
-    if (len & 1) pad += ch;
+    // add \`ch\` to \`npm\` if \`len\` is odd
+    if (len & 1) npm += ch;
     // divide \`len\` by 2, ditch the remainder
     len >>= 1;
     // \\"double\\" the \`ch\` so this operation count grows logarithmically on \`len\`
@@ -47,6 +47,6 @@ function leftPad (str, len, ch) {
     // \`len\` is 0, exit the loop
     else break;
   }
-  // pad \`str\`!
-  return pad + str;
+  // npm \`str\`!
+  return npm + str;
 }
END SNAPSHOT"
`;

exports[`Test lockfile: check patch with lockfile 3 1`] = `
"SNAPSHOT: check patch with lockfile 3
diff --git a/node_modules/left-pad/index.js b/node_modules/left-pad/index.js
index e90aec3..7f8fdaa 100644
--- a/node_modules/left-pad/index.js
+++ b/node_modules/left-pad/index.js
@@ -22,9 +22,9 @@ var cache = [
 function leftPad (str, len, ch) {
   // convert \`str\` to a \`string\`
   str = str + '';
-  // \`len\` is the \`pad\`'s length now
+  // \`len\` is the \`npm\`'s length now
   len = len - str.length;
-  // doesn't need to pad
+  // doesn't need to npm
   if (len <= 0) return str;
   // \`ch\` defaults to \`' '\`
   if (!ch && ch !== 0) ch = ' ';
@@ -32,12 +32,12 @@ function leftPad (str, len, ch) {
   ch = ch + '';
   // cache common use cases
   if (ch === ' ' && len < 10) return cache[len] + str;
-  // \`pad\` starts with an empty string
-  var pad = '';
+  // \`npm\` starts with an empty string
+  var npm = '';
   // loop
   while (true) {
-    // add \`ch\` to \`pad\` if \`len\` is odd
-    if (len & 1) pad += ch;
+    // add \`ch\` to \`npm\` if \`len\` is odd
+    if (len & 1) npm += ch;
     // divide \`len\` by 2, ditch the remainder
     len >>= 1;
     // \\"double\\" the \`ch\` so this operation count grows logarithmically on \`len\`
@@ -47,6 +47,6 @@ function leftPad (str, len, ch) {
     // \`len\` is 0, exit the loop
     else break;
   }
-  // pad \`str\`!
-  return pad + str;
+  // npm \`str\`!
+  return npm + str;
 }
END SNAPSHOT"
`;
